docs/active-job.md

```diff
@@ -0,0 +1,40 @@
+# Rails Active Job
+
+We use Rails [Active Job](https://guides.rubyonrails.org/active_job_basics.html) to process asynchronous workloads. Active Job supports various backends, or 'queue adapters', to execute jobs. We use one called [GoodJob](https://github.com/bensheldon/good_job). GoodJob is similar to the more widely used [Sidekiq](https://github.com/sidekiq/sidekiq), except it uses the existing Postgres database rather than requiring a Redis instance. This means we can keep our stack leaner and save on cloud infrastructure costs.
+
+## Admin dashboard
+
+Access the GoodJob admin dashboard here:  
+https://manage-school-placements-qa.test.teacherservices.cloud/good_job
```

Comment:
It's good to note that this is for the QA environment. We should also add the https://track-and-pay-qa.test.teacherservices.cloud/good_job link.

Reply:
I didn't include a link to both services because they will both show the same information, since they both run on the same app & database instances.

But I'll add them for completeness.

Reply:
@Nitemaeric I've [just updated](https://github.com/DFE-Digital/itt-mentor-services/compare/cd59f9dd874a8518253be0a472fc4301275529e0..ed0763e92349e7063db8e63da8b64274adc41f01) the doc to include both links.